src/formaction.cpp

```diff
@@ -174,7 +174,7 @@ bool FormAction::process_op(Operation op,
 		v->goto_prev_dialog();
 		break;
 	default:
-		this->process_operation(op, automatic, args);
+		return this->process_operation(op, automatic, args);
```

Comment:
Man I wish we were on C++17 already and could mark that function `[[nodiscard]]` (or in Rust, returning `Result` or something).